meta registration of linalg_inv_ex

avikchaudhuri opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #92021
* #92020
* __->__ #92019

Closely following D41488040.

Differential Revision: [D42456011](https://our.internmc.facebook.com/intern/diff/D42456011/)

avikchaudhuri committed:
meta registration of linalg_inv_ex

Closely following D41488040.

Differential Revision: [D42456011](https://our.internmc.facebook.com/intern/diff/D42456011/)

[ghstack-poisoned]

SherlockNoMad reviewed (approved):
A few more xfail test cases can be removed now.

avikchaudhuri committed:
Update on "meta registration of linalg_inv_ex"


Closely following D41488040.

Differential Revision: [D42456011](https://our.internmc.facebook.com/intern/diff/D42456011/)

[ghstack-poisoned]

avikchaudhuri committed:
Update on "meta registration of linalg_inv_ex"


Closely following D41488040.

Differential Revision: [D42456011](https://our.internmc.facebook.com/intern/diff/D42456011/)

[ghstack-poisoned]
